### DISCUSSION TITLE
Contributing: Is the project open to any idea?

### DISCUSSION THREAD

**Discussion by herronelou** [themesberg/flowbite-svelte#1257] [General]
Hello.

I'm using flowbite svelte, and while it does well in most situations, I often run into components that are nearly doing what I need but not quite.
The big one that happened yesterday, while using a MultiSelect, my number of options has grown so much that the MultiSelect has become inconvenient to find the desired option, and as such I'm about to either remake my own version of it and add an input for typing, or use another library such as https://multiselect.janosh.dev/ and style it to look like flowbite.

I've not been involved with any of the community so I'm not sure if contributions are open for any idea, for example, I could contribute back my improved multi select, but I know some other open source projects I've used can be picky, and can sometimes reject features as being out of scope. I'm also not too clear on the relationship between flowbite-svelte and flowbite. Would any new feature of flowbite-svelte also need to be ported to flowbite (are they intended to have feature parity?)?

**Comment by shinokada** [comment]
We have accepted some community components in the past. See https://flowbite-svelte.com/docs/extend/marquee, and https://flowbite-svelte.com/docs/extend/step-indicator.
They are not in Flowbite.com, but our community members thought it will be useful to be added to Flowbite-Svelte.
If is useful and adapting the same style, we can add it under the Extend (Community) sub-menu. So yes, we are open to a contribution.

**Reply by herronelou** [reply]
Thanks for the reply.
Looking into fixing the MultiSelect has exposed a few bugs, so I'll be opening an issue. I'm trying to fix them myself but I'm a python dev, fairly new to JS/Svelte, so unsure if my approach will be good enough for the project.
